dotnet-desktop-guide/net/winforms/overview/index.md

```diff
@@ -32,6 +32,10 @@ With Windows Forms, you develop graphically rich apps that are easy to deploy, u
 
 To learn how to create a Windows Forms app, see [Tutorial: Create a new WinForms app (Windows Forms .NET)](../get-started/create-app-visual-studio.md).
 
+## Why migrate from .NET Framework
+
+Windows Forms for .NET 5.0 provides new features and enhancements over .NET Framework. For more information, see [What's new (Windows Forms.NET)](../whats-new/index.md). To learn how to migrate an app, see [How to migrate a Windows Forms desktop app to .NET 5](../migration/index.md).
```

Comment:
Is `Windows Forms.NET` an official term? It looks very odd to me. Should it be `Windows Forms .NET`?

Reply:
I suggest "Windows Forms for .NET 5", similar to the following link text "migrate ... to .NET 5"

Reply:
Yes, it's supposed to be spaced correctly.

Reply:
Is it ".NET 5.0" or just ".NET 5"? 🤔

Reply:
But in general I like it "[How to migrate a Windows Forms desktop app to .NET]"